src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -223,6 +223,18 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 acrylic.BackgroundSource(Media::AcrylicBackgroundSource::HostBackdrop);
             }
 
+            // Initialize background color so we don't get a

```

Comment:
```suggestion
            // see GH#1082: Initialize background color so we don't get a
```
This seems like a little bit less-obvious bug fix, so referring back to the original issue might help people get some context on why it was made originally.